data/reusables/actions/expression-syntax-if.md

```diff
@@ -1 +1 @@
-When you use expressions in an `if` conditional, you may omit the {% raw %}`${{ }}`{% endraw %} expression syntax because {% data variables.product.prodname_actions %} automatically evaluates the `if` conditional as an expression. Using the {% raw %}`${{ }}`{% endraw %} expression syntax turns the contents into a string, and strings are truthy. For example, `if: true && {% raw %}${{ false }}{% endraw %}` will evaluate to `true`.
+When you use expressions in an `if` conditional, you may omit the {% raw %}`${{ }}`{% endraw %} expression syntax because {% data variables.product.prodname_actions %} automatically evaluates the `if` conditional as an expression. You can't omit the {% raw %}`${{ }}`{% endraw %} expression syntax when expression starts with {% raw %}`!`{% endraw %}, it's a limitation of YAML format. Using the {% raw %}`${{ }}`{% endraw %} expression syntax turns the contents into a string, and strings are truthy. For example, `if: true && {% raw %}${{ false }}{% endraw %}` will evaluate to `true`.
```

Comment:
```suggestion
When you use expressions in an `if` conditional, you may omit the {% raw %}`${{ }}`{% endraw %} expression syntax because {% data variables.product.prodname_actions %} automatically evaluates the `if` conditional as an expression. You must use the {% raw %}`${{ }}`{% endraw %} expression syntax when the expression starts with `!`, since `!` is reserved notation in YAML format. Using the {% raw %}`${{ }}`{% endraw %} expression syntax turns the contents into a string, and strings are truthy. For example, `if: true && {% raw %}${{ false }}{% endraw %}` will evaluate to `true`.
```

Reply:
Small suggestion for tone, let me know what you think. 

I think you can omit the `{% raw %}` tag around `!`. In case you're curious (and while we are on the topic of notation 😆), we use the `{% raw %}` tag to print `{` onto the docs site because `{` is a special character in the Liquid syntax, which we use for [things like callouts](https://docs.github.com/en/contributing/syntax-and-versioning-for-github-docs/using-markdown-and-liquid-in-github-docs).